testdata/cheats/Vm.sol

```diff
@@ -277,6 +277,7 @@ interface Vm {
     function getNonce(address account) external view returns (uint64 nonce);
     function getNonce(Wallet calldata wallet) external returns (uint64 nonce);
     function getRecordedLogs() external returns (Log[] memory logs);
+    function getStateDiff(bool json) external view returns (string memory diff);
```

Comment:
Style nit: for consistency it may be preferred to split this into `getStateDiff` and `getStateDiffJson`

This is in line with entries like `writeJson`, `keyExistsJson`, etc..

This makes it easier to add alternative formatting modes in the future if required

Reply:
updated in https://github.com/foundry-rs/foundry/pull/9435/commits/4e684f8eca67ad28e3c7d85e53706581907f189f